lib/vsc/mympirun/option.py

```diff
@@ -82,7 +83,11 @@ def make_init(self):
             "double": ("Run double the amount of processes (eg for GAMESS; to change multiplier, use --hybrid)",
                        None, "store_true", False),
 
-            "output": ("filename to write stdout/stderr directly to (instead of stdout)", "str", "store", None),
+            "output": ("redirect the output of mpirun to a file (instead of stdout/stderr)",
```

Comment:
Can you add a log.info prior to execution of mpirun that repeats to where it will output?